CONTRIBUTING.md

```diff
@@ -1,41 +1,57 @@
-## Contributions Welcome!
+# Contributions Welcome!
 
 Interested in contributing? We'd love your help!
 
 Our [issue tracker](https://github.com/pikapkg/snowpack/issues) is always organized with a selection of high-priority bugs, feature requests, and ["help wanted!"](https://github.com/pikapkg/snowpack/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)/["good first issue"](https://github.com/pikapkg/snowpack/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) items. For general package troubleshooting and discussions, check out our [Package Community](https://www.pika.dev/npm/snowpack/discuss) discussion board.
 
+## Requirements
+
+You need [yarn](https://classic.yarnpkg.com/en/docs/install) to setup the `snowpack` repository locally on your machine
```

Comment:
Might want to note you can use another similar package manager like npm if you want (I think??). I'd imagine most people who could contribute know that, but just in case someone thinks you absolutely need yarn.

Reply:
We actually use yarn's workspaces feature to manage the multiple packages in the monorepo, so in this case you actually do need yarn. @gr2m it would be good to explain that quickly here, in case others have questions or assume it might just be optional.

Reply:
Maybe:

"Snowpack uses [yarn workspaces](https://classic.yarnpkg.com/) to manage the multiple packages contianed in this repo. To contribute, [make sure that you have Yarn installed on your machine.]((https://classic.yarnpkg.com/en/docs/install)